### ISSUE TITLE
Implement early stopping

### ISSUE THREAD

**Issue by ThomasHegelund** [PBonvang/fundus-image-classifier#1]
Implement a function that looks at the training and validation loss and stops the training if the model starts to overfit.

[This](https://medium.com/analytics-vidhya/early-stopping-with-pytorch-to-restrain-your-model-from-overfitting-dce6de4081c5) could be a useful source for help
